### ISSUE TITLE
Terms which seem to not be fully connected to parent terms

### ISSUE THREAD

**Issue by seger** [obophenotype/xenopus-phenotype-ontology#128]
Via @malcolmfisher103:

We have just finished revising our scripts for mapping our old legacy data to XPO terms. One of the last things we did was add a step in our mapping scripts to reduce multiple possible mappings that were in the same 'path' in the ontology down to the single lowest level term. In this process we identified a number of terms which seemed to not be fully connected to parent terms as we might expect. 
 
As an example, an annotation was mapped to both XPO:0141561 'abnormal tibia-fibula morphology in the regenerating hindlimb' and XPO:0102813 'abnormal regenerating hindlimb morphology'. Both of these are correct but I would expect 'abnormal tibia-fibula morphology in the regenerating hindlimb' to be a child of 'abnormal regenerating hindlimb morphology' and the parent term to have been removed with our reduction step. We find 'abnormal tibia-fibula morphology in the regenerating hindlimb' to be a direct child of 'abnormal regenerating hindlimb' instead. Is the expectation not that abnormal morphology of a part means abnormal morphology of the whole?

**Comment by seger** [comment]
@matentzn Nico, we seem to recall that there have been discussions about this in the past. Could you refresh us on the reasoning behind it? Thanks!

**Comment by matentzn** [comment]
Yes; this is a long-standing discussion: The fear was always the "granularity-overload" -> if you apply the pattern with the including taking the parts into account, you can get that `"abnormal morphology of atom" ---[implies]-->  "abnormal morphology of whole organism"`. Now because your use case is so important, we have decided to add the abnormal morphology pattern that also takes parts into account: https://github.com/obophenotype/upheno/tree/master/src/patterns/dosdp-patterns#abnormal-morphology-of-part-of-anatomical-entity. I leave this up to you to work with, but I would advice (weakly) against using this pattern as the rule, and preferring the stricter pattern (without the parts) instead. I don't want the granularity issues to propagate everywhere across upheno, if at all possible, but yeah, decide internally which way you want to go! If its a big deal, just switch to the other pattern.
